.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Debug Jest Tests on Nix",
```

Comment:
I was able to step through jest tests with this.  It works on mac / linux and I saw mention of different configs for windows but I didn't do that yet or validate so I called Nix.